docs/pages/blog/mui-material-v5-3.md

```diff
@@ -0,0 +1,222 @@
+
+## Using callback in `styleOverrides`
+
+The callback gives you the `props` that the slot received. Most of the time, you would use:
+
+- `props.ownerState`: the combination of runtime props and internal states.
+- `props.theme`: the theme you provide to `<ThemeProvider theme={yourTheme}>` or the default theme.
+
+```jsx
+import { ThemeProvider, createTheme } from '@mui/material/styles';
+
+<ThemeProvider
+  theme={createTheme({
+    components: {
+      MuiChip: {
+        styleOverrides: {
+          // you can now use the theme without creating the initial theme!
+          root: ({ ownerState, theme }) => ({
+            padding: {
+              small: '8px 4px',
+              medium: '12px 6px',
+              large: '16px 8px',
+            }[ownerState.size],
+            ...(ownerState.variant === 'outlined' && {
+              borderWidth: '2px',
+              ...(ownerState.variant === 'primary' && {
+                borderColor: theme.palette.primary.light,
+              }),
+            }),
+          }),
+          label: {
+            padding: 0,
+          },
+        },
+      },
+    },
+  })}
+>
+  ...your app
+</ThemeProvider>;
+```
+
+> 💡 The side benefit of using a callback is that you can use the theme without creating the outer scoped variable.
+
+### Typescript
+
+The callback is typed-safe. `ownerState` is the `ComponentProps` interface and `theme` is `Theme` interface.
+
+```tsx
+{
+  MuiChip: {
+    styleOverrides: {
+      // ownerState: ChipProps
+      // theme: Theme
+      root: ({ ownerState, theme }) => ({...}),
+    },
+  }
+}
+```
+
+If you extend the `ComponentProps` or [`Theme`](/customization/theming/#custom-variables) via module augmentation, you will be able to see them inside the callback 🎉.
+
+## Experimental `sx` function
+
+Initially, `sx` is designed to be a prop that you can inject style with shorthand notation to the component created by `styled` API:
+
+```jsx
+import { styled } from '@mui/material/styles';
+import Box from '@mui/material/Box';
+
+const Label = styled('span')({
+  fontWeight: 'bold',
+  fontSize: '0.875rem',
+})
+
+<Box sx={{ display: 'flex' }}>
+  <Label sx={{ color: 'text.secondary' }}>Label</Label>
+</Box>;
+```
+
+> 💡 All MUI components are created by `styled` API, so they accept `sx` prop by default.
+
+`sx` helps developers write less code and be more productive once they are familiar with the API. With the callback support in `styleOverrides`, it is now possible to use `sx` like syntax at the global theme overrides.
+
+All you need to do is using the [`experimental_sx`](/system/styled/#how-can-i-use-the-sx-syntax-with-the-styled-utility) function:
+
+```jsx
+import {
+  ThemeProvider,
+  createTheme,
+  experimental_sx as sx,
+} from '@mui/material/styles';
+
+<ThemeProvider
+  theme={createTheme({
+    components: {
+      MuiChip: {
+        styleOverrides: {
+          root: sx({
+            px: 1.5, // shorthand for padding-left & right
```

Comment:
I would use here `px` value, as we are not describing that this is multiplied by the theme spacing. For more details, developers can take a look at the other links.